railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
Rails users are still suffering a slower `bundle install` and given you run `rails new` way less than `bundle install` I think we are fixing anything with this, just adding complexity to the framework.